code_samples/project_model/README.md

```diff
@@ -0,0 +1,37 @@
+# Project Model Sample [![JetBrains IntelliJ Platform SDK Docs](https://jb.gg/badges/docs.svg)][docs]
+*Reference: [PyCharm Plugin Development in IntelliJ SDK Docs][docs:pycharm]*
+
+## Quickstart
+
+Project Model Sample project provides five actions that present data extracted using `ProjectRootManager` instance
```

Comment:
Suggest: _The Project Model Sample project..._ 
and _rename the SDK used in the project,_